plugins/net_plugin/net_plugin.cpp

```diff
@@ -3843,6 +4003,22 @@ namespace eosio {
             my->chain_plug->enable_accept_transactions();
          }
 
+         //if we have certificate option that TLS must be enabled
+         if ( options.count("p2p-tls-own-certificate-file") ) {
+            auto certificate = options["p2p-tls-own-certificate-file"].as<string>();
+            auto pkey        = options["p2p-tls-private-key-file"].as<string>();
+            auto ca_cert     = options["p2p-tls-security-group-ca-file"].as<string>();
+
```

Comment:
need to add this here:
            auto relative_to_absolute = [](bfs::path& file) {
               if( file.is_relative()) {
                  file = bfs::current_path() / file;
               }
            };
            relative_to_absolute(certificate);
            relative_to_absolute(pkey);
            relative_to_absolute(ca_cert);

to turn relative paths into absolute paths